database/seeders/TestDataSeeder.php

```diff
@@ -687,6 +688,8 @@ public function run(): void
             ],
         ];
 
+        $inkind = PaymentType::factory(['name' => ['en' => 'In-kind donation']])->create();
+
```

Comment:
Does this need to be in the TestDataSeeder? It's adding in an extra payment type when the TestData seeder is run which happens anytime the DevSeeder is run (usually local and dev environments). I think this may be confusing as it isn't an actual payment type in the system.